interactions/api/events/processors/voice_events.py

```diff
@@ -13,15 +13,20 @@
 class VoiceEvents(EventMixinTemplate):
     @Processor.define()
     async def _on_raw_voice_state_update(self, event: "RawGatewayEvent") -> None:
-        before = copy.copy(self.cache.get_voice_state(event.data["user_id"])) or None
         after = await self.cache.place_voice_state_data(event.data)
+        is_bot = event.data["member"]["user"]["bot"]
```

Comment:
This should likely be checking if the user is the specific bot running, rather than any bot. `bot_voice_state` is really only meant for the running bot - yes, this means the voice state implementation is flawed as it doesn't handle other bots correctly, there's not much we can do besides for a breaking release 😅.

Reply:
Changed this to something similar to how it was done before -- checking if `self.user.id` is the same as the event user id